shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Results",
```

Comment:
the results object i created some time ago had a boolean for success/fail and a message. I think this is what we need. in my opinion the enum only complicates things.

Reply:
After a follow-up discussion, let's investigate how we can reuse instance's `error` field. If there is no simple way, we will need the `completedWith`. Otherwise, let's remove the `compledWith` and use just the `message` property.